src/platforms/rp2040/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ option(AVM_DISABLE_SMP "Disable SMP support." OFF)
 option(AVM_USE_32BIT_FLOAT "Use 32 bit floats." OFF)
 option(AVM_VERBOSE_ABORT "Print module and line number on VM abort" OFF)
 option(AVM_WAIT_FOR_USB_CONNECT "Wait for USB connection before starting" OFF)
+option(AVM_WAIT_BOOTSEL_ON_EXIT "Wait in BOOTSEL rather than shutdown on exit" OFF)
```

Comment:
I left this off in this PR, but I really think this should be on by default. It makes flashing new applications much easier, and the only downside is possibly a little more power consumption after the application exits, but I do not know how much power is saved when the processor is hung.

Reply:
I was going to ask the same, making this the default seems a good idea.

Reply:
This feature is now enabled by default.